### DISCUSSION TITLE
Add Support for GitHub Models

### DISCUSSION THREAD

**Discussion by GreeningSiren** [open-webui/open-webui#5123] [General]
**Describe the solution you'd like**

Make a way to use GitHub Models for use in open-webui

**Comment by Peter-De-Ath** [comment]
What are GitHub Models? Could you provide any links/examples?

**Comment by GreeningSiren** [comment]
https://docs.github.com/en/github-models
https://github.com/marketplace/models

A service from GitHub that lets you talk to and experiment with AI models for free.
